performance-timeline/po-entries-sort.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE HTML>
+<head>
+<meta charset="utf-8">
+<title>PerformanceObservers: getEntries* sort order</title>
+<meta name="author" title="JosephPecoraro" href="mailto:joepeck@webkit.org">
+<meta name="assert" content="PerformanceObserverEntryList entries are sorted correctly">
+<link rel="help" href="https://w3c.github.io/performance-timeline/#filter-performance-entry-buffer-by-name-and-type">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="performanceobservers.js"></script>
+</head>
+<h1>PerformanceObservers: getEntries* sort order</h1>
+<p>getEntries, getEntriesByType and getEntriesByName sort order</p>
+<div id="log"></div>
+<script>
+  async_test(function (t) {
+    var stored_entries = [];
+    var stored_entries_by_type = [];
+    var observer = new PerformanceObserver(
+        t.step_func(function (entryList, obs) {
+
+          stored_entries = entryList.getEntries();
+          stored_entries_by_type = entryList.getEntriesByType("mark");
+          stored_entries_by_name = entryList.getEntriesByName("name-repeat");
+
+          checkSorted(stored_entries);
+          checkEntriesInOrder(stored_entries, [
```

Comment:
Based on the 2/8 meeting this test should be less strict. Namely just guarantee start time sorted, no guaranteed order for entries with equivalent start times.